core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt

```diff
@@ -7,11 +7,11 @@ import kotlinx.datetime.TimeZone
 import kotlinx.datetime.toInstant
 import kotlinx.datetime.toLocalDateTime
 
-public enum class DroidKaigi2023Day(
-    public val day: Int,
-    public val dayOfMonth: Int,
-    public val start: Instant,
-    public val end: Instant,
+enum class DroidKaigi2023Day(
```

Comment:
Maybe we are going to use explicit mode. But we can fix it later 🙏

Reply:
Sorry, on second thoughts. Could you revert this change..? 🙇

Reply:
Sorry too. I fixed it.
[3240b94](https://github.com/DroidKaigi/conference-app-2023/pull/818/commits/3240b9428161e75c0bf5d2cd5ee4aa089d8a5ba7)